Design improvements

caleb531 opened the pull request:
Hi, Jackson. I've made a number of improvements to your blog:

- Use Gravatar for favicon and apple-touch-icon
- Make header Gravatar retina-ready
- Move pages to _pages directory
- Made homepage a Markdown file so content can be added
- Add site footer with copyright notice

jacksonrayhamilton commented:
Thanks! The 404 page doesn't seem to work anymore. Also, I would rather avoid using a footer, as I don't think it fits the minimalist theme. Copyright is automatic, so the notice is not necessary either. Otherwise, LGTM.

caleb531 commented:
Alright, I fixed the 404 page (it was simply building to the wrong location—it now builds to 404.html in the root of the build directory). I also reverted the site footer as you asked. :)
